src/app/chat-list-item/chat-list-item.component.html

```diff
@@ -0,0 +1,27 @@
+<button
+  *ngIf="chatMetadata"
+  mat-list-item
+  class="h-auto my-2"
+  (click)="onClick()"
+>
+  <div class="container">
+    <div class="row">
+      <div class="col-auto p-0">
+        <img
+          class="rounded-circle"
+          height="56px"
+          [src]="chatMetadata.groupImageUrl"
+        />
+      </div>
+      <div class="col">
+        <span>{{ chatMetadata.groupName }}</span> <span>Time</span>
```

Comment:
is the <span>Time</span> suppose to be a place holder? Because right now when you run the code it shows, for example a group called Horse Club

Horse ClubTime

Reply:
This should be formatted and date piped. Will do along with merging main into this branch.